src/pal/src/exception/seh-unwind.cpp

```diff
@@ -97,6 +97,28 @@ static void WinContextToUnwindCursor(CONTEXT *winContext, unw_cursor_t *cursor)
     unw_set_reg(cursor, UNW_X86_64_R14, winContext->R14);
     unw_set_reg(cursor, UNW_X86_64_R15, winContext->R15);
 #elif defined(_ARM_)
+#ifdef __LINUX__
```

Comment:
I think we should make this change unconditional - not just for Linux. Unless we start using the LLVM libunwind for some ARM systems, we will use the same libunwind everywhere and so the behavior will be the same.